openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/dweet-io.js

```diff
@@ -0,0 +1,520 @@
+//     dweet.io.js
+//     http://dweet.io
+//     (c) 2014 Jim Heising and Bug Labs, Inc.
+//     dweet.io.js may be freely distributed under the MIT license.
+(function () {
+
+	var isNode = true;
+
+	// Is this loading into node.js?
+	try {
+		isNode = (require);
+	}
+	catch (e) {
+		isNode = false;
+	}
+
+	var io;
+	var request;
+
+	var LAST_THING_NAME = "last-thing.dat";
+	var DWEET_SERVER = "https://dweet.io:443";
```

Comment:
I doubt connection to external server will be allowed by CSP

Reply:
During our  testing  on a  live  site we did not  experience any  issues with this.